spec/GraphQLOverHTTP.md

```diff
@@ -738,10 +734,15 @@ Note: The GraphQL specification
 and refers to the situation wherein a _GraphQL field error_ occurs as a partial
 response; it still indicates successful execution.
 
-## Processing the response
+## Processing a response
+
+The source of a response may be an intermediary server, such as an API gateway,
+proxy, or firewall, in certain environments.
+
+Those intermediary servers may not understand GraphQL: in the case of an error,
+they may return their own non-GraphQL `application/json` response with a
+non-`200` status code.
```

Comment:
I'd still recommend removing the 'may not understand GraphQL' part.  I'm good with the rest of this PR.

```suggestion
In certain environments, the source of a response may be an intermediary server,
such as an API gateway, proxy, or firewall. In the case of an error, they may return
their own non-GraphQL `application/json` response with a non-`200` status code.
```